tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,190 @@
+    )
+    def test_otel_traces_traceidratio(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_sample_rate"] == "0.1"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_always_on",}],
+    )
+    def test_otel_traces_parentbased_on(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_sample_rate"] == "1.0" or "1"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_always_off",}],
+    )
+    def test_otel_traces_parentbased_off(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_sample_rate"] == "0.0" or "0"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_traceidratio", "OTEL_TRACES_SAMPLER_ARG": "0.1"}],
+    )
+    def test_otel_traces_parentbased_ratio(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_sample_rate"] == "0.1"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_EXPORTER": "none"}],
+    )
+    def test_otel_traces_exporter_none(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_enabled"] == "false"
+
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_LOG_LEVEL": "debug"}],
+    )
+    def test_otel_log_level_to_debug_mapping(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_log_level"] == "debug"
+        assert resp["dd_trace_debug"] == "true"
+
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"DD_TRACE_OTEL_ENABLED": "true", "OTEL_SDK_DISABLED": "true"}],
+    )
+    def test_dd_trace_otel_enabled_takes_precedence(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_otel_enabled"] == "true"
+
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_SDK_DISABLED": "true"}],
+    )
+    def test_otel_sdk_disabled_set(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_otel_enabled"] == "true"
```

Comment:
Setting `OTEL_SDK_DISABLED` to true should turn off `DD_TRACE_OTEL_ENABLED`, right?
```suggestion
        assert resp["dd_trace_otel_enabled"] == "false"
```